texas-hearing-institute/App.tsx

```diff
@@ -1,21 +1,70 @@
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { Pressable, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import { Button } from 'react-native-elements';
 import tw from 'tailwind-react-native-classnames';
+import Icon from 'react-native-vector-icons/FontAwesome';
 
+
+// LOOK AT -TTS
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text style={tw`text-3xl font-bold underline`}>Open up App.tsx to start working on your app!</Text>
+    <View style={[styles.container, 
+    {
+      flexDirection: 'column',
```

Comment:
since you only use `styles.container` for this single View, you can go ahead and add the flexDirection to the container property in the stylesheet!